vue-components/README.md

```diff
@@ -24,6 +26,16 @@ or in your entry component, e.g `App.vue`, add
 
 `import { Button } from '@wmde/wikit-vue-components';`
 
+### Vue 3 migration build
+
+There is an experimental build that works with the Vue 3 migration build (`@vue3/compat`).
+To use it, change your imports to
+`@wmde/wikit-vue-components/dist/wikit-vue-components-vue3compat.common.js`
+(or configure `@wmde/wikit-vue-components` to be an alias for that),
```

Comment:
The following Vite config snippet seems to work for that (in one of the PoC changes):

```js
resolve: {
    alias: {
        '@wmde/wikit-vue-components/dist/wikit-vue-components.css': path.resolve( __dirname, 'node_modules/@wmde/wikit-vue-components/dist/wikit-vue-components.css' ),
        '@wmde/wikit-vue-components': path.resolve( __dirname, 'node_modules/@wmde/wikit-vue-components/dist/wikit-vue-components-vue3compat.common.js' ),
    },
},
```